gateway-js/src/__tests__/integration/networkRequests.test.ts

```diff
@@ -130,7 +132,11 @@ it('Fetches Supergraph SDL from remote storage using a configured env variable',
 
 it('Updates Supergraph SDL from remote storage', async () => {
   mockSupergraphSdlRequestSuccess();
-  mockSupergraphSdlRequestSuccess(getTestingSupergraphSdl(fixturesWithUpdate), 'updatedId-5678');
+  mockSupergraphSdlRequestSuccessIfAfter(
+      'originalId-1234',
+    'updatedId-5678',
+      getTestingSupergraphSdl(fixturesWithUpdate),
+  );
```

Comment:
Indentation consistency nit:
```suggestion
  mockSupergraphSdlRequestSuccessIfAfter(
    'originalId-1234',
    'updatedId-5678',
    getTestingSupergraphSdl(fixturesWithUpdate),
  );
```

Reply:
yeah, IntelliJ is too smart for my own good with those little field labels it display.